tests/test_library.py

```diff
@@ -74,8 +74,9 @@ def test_get_images_for_unsupported_uri_type_issues_warning(config, caplog):
     search_uri = PandoraUri.factory("pandora:search:R12345")
     results = backend.library.get_images([search_uri.uri])
     assert len(results[search_uri.uri]) == 0
+    print(f"TEXT: {caplog.text}")
```

Comment:
Can we remove this `print` statement? I assume it was used for debugging purposes.

If it would be sensible to make this an assertion to be included in the test that would be even better.

Reply:
Yep, we can remove the print statement, I really didn't clean up my code did I!